Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -798,14 +798,34 @@ def perform_rasterize(path: str | list[str],
     :param width: window width
     :param height: window height
     """
+
+    # convert the path param to list in case we have only one string
+    path = [path] if isinstance(path, str) else path
+
+    # create a list with all the paths starts with "mailto:"
+    mailto_paths = [path_value for path_value in path if path_value.startswith('mailto:')]
+
+    if mailto_paths:
+        # remove the mailto from path param
+        path = list(set(path) - set(mailto_paths))
+        demisto.error(f'perform_rasterize skip the following paths: {mailto_paths}')
+        return_results(CommandResults(
+            readable_output=f'URLs that start with "mailto:" cannot be rasterized.\nURL: {mailto_paths}'))
```

Comment:
We want to throw the whole rasterization?

Reply:
why not?

Reply:
i would also think that if some of the paths are valid, it should perform the rasterization for those. 
other thing, in this situation when its not perform rasterization for the valid paths, line 810 looks redundant right? because the function returned at 812